cms/templates/widgets/problem-edit.html

```diff
@@ -159,4 +163,15 @@ <h6>${_("Explanation")}</h6>
     </article>
 </script>
 </div>
+<div id="visual-editor" class="visual-editor is-inactive"></div>
+<script>
+    var metaData = ${editable_metadata_fields| n, dump_js_escaped_json};
+    window.webpackJsonp=[]; // built-in webpack is not Array, react app expects an Array
```

Comment:
We should not be messing with the `window.webpackJsonp` global. It will break other things;
![Screen Shot 2019-08-13 at 3 20 20 PM](https://user-images.githubusercontent.com/945577/62981461-22751a80-bdde-11e9-8cdb-ee14d9a02300.png)


I believe the difference here is that webpack 4 uses an array, but webpack 3 uses a function.

How did you build `bundle.js` ? I looked at https://github.com/open-craft/capa-visual-editor but I don't see any `webpack` configuration/code at all. I believe you need to change the `output.jsonpFunction` webpack setting to something else like `visualEditorWebpackJsonp` and then it should fix the conflict. But in either case, can you please include the webpack/build code in the `capa-visual-editor` repo?

Reply:
Assembled it with webpack - now all is being gathered well without errors